addons/l10n_cl/models/account_move.py

```diff
@@ -161,6 +161,36 @@ def _l10n_cl_get_invoice_totals_for_report(self):
             tax_totals['formatted_amount_untaxed'] = formatLang(self.env, tax_totals['amount_untaxed'], currency_obj=self.currency_id)
             if tax_totals['subtotals']:
                 tax_totals['subtotals'][0]['formatted_amount'] = tax_totals['formatted_amount_untaxed']
+        values = self._l10n_cl_get_amounts()
```

Comment:
In this last commit, instead of modifying the totals section with a new inherited template, the idea is to use the pre-existing method `_l10n_cl_get_invoice_totals_for_report()` and add the missing information to that dictionary. The final result is like this:

<img width="614" alt="Captura de pantalla 2023-10-03 a la(s) 13 03 10" src="https://github.com/odoo/odoo/assets/382893/a863f70e-8ba6-43b8-a02b-26206796428e">